tools/VscExtension/package.json

```diff
@@ -1,8 +1,8 @@
 {
-  "name": "DocFXPreview",
-  "displayName": "DocFXPreview",
-  "description": "Preview Html and token tree generated by DocFX in Visual Studio Code",
-  "version": "0.0.24",
+  "name": "DocFX",
+  "displayName": "DocFX",
+  "description": "An amazing tool to make community users using and contributing to DocFX easier and more convenient, the main features include previewing DocFX Flavored Markdown with different templates, generating token tree of markdown file",
```

Comment:
doesn't have now, but we should have it as advanced user manual

---
In reply to: [131837142](https://github.com/dotnet/docfx/pull/1936#discussion_r131837142) [](ancestors = 131837142)